src/Data/FormURLEncoded.purs

```diff
@@ -27,15 +28,11 @@ toArray :: FormURLEncoded -> Array (Tuple String (Maybe String))
 toArray (FormURLEncoded a) = a
```

Comment:
I think we should add a `Newtype` instance, but still have monophonic functions.

I think `Newtype` is very nice with it's higher-order combinators (e.g. ``over MyNewType f`` vs `MyNewtype <<< f <<< runNewType`), and possibility for abstracting over newtypes (e.g. `StrMap` with polymorphic functions `forall key. Newtype key String`).

However for simple construction/deconstruction I think I'll often be preferring plain functions:
  - they make mistakes more wrong looking, in order to construct/deconstruct a type you have to somehow mention it's name, either directly in a function name (`runNewtype`) or through qualified import (`MyNewType.fromArray`).
  - they are more interference-friendly `(encode $ FormURLEncoded.fromArray [...]` vs `encode (wrap [ ... ] :: FormUrlEncoded)`.